src/trivia/util.h

```diff
@@ -148,6 +148,16 @@ strnindex(const char *const *haystack, const char *needle, uint32_t len,
 #  define __has_cpp_attribute(x) 0
 #endif
 
+/**
+ * Use instead of `*ptr = value` statement in case ptr is unaligned and
+ * compiler is not informed of unalignment.
+ */
+#define uptr_deref_set(ptr, value) \
```

Comment:
Let's rename to `unaligned_store`? I think we also need `unaligned_load` so that we can drop the pragmas.

Reply:
Same as above. Not applicable anymore.